src/components/v5/common/Pills/PillsBase.tsx

```diff
@@ -32,7 +33,7 @@ const PillsBase: FC<PropsWithChildren<PillsProps>> = ({
       {Icon && (
         <span className="flex shrink-0">
           <Icon
-            size={pillSize === 'medium' ? 14 : 12}
+            size={iconSize || (pillSize === 'medium' ? 14 : 12)}
```

Comment:
Is there anything speaking against using the `small` value for the `pillSize` property?